src/Shared/Components/CICDHistory/utils.tsx

```diff
@@ -137,3 +140,59 @@ export const getStageStatusIcon = (status: StageStatusType): JSX.Element => {
             return <ICInProgress className="dc__no-shrink icon-dim-16 ic-in-progress-orange" />
     }
 }
+
+const renderAbortedTriggerIcon = (): JSX.Element => <ICAborted className="icon-dim-20 dc__no-shrink" />
+const renderFailedTriggerIcon = (): JSX.Element => <ICFailure className="icon-dim-20 dc__no-shrink" />
+const renderProgressingTriggerIcon = (): JSX.Element => (
+    <ICInProgress className="dc__no-shrink icon-dim-20 ic-in-progress-orange" />
+)
+const renderSuccessTriggerIcon = (): JSX.Element => (
+    <div className="dc__app-summary__icon dc__no-shrink icon-dim-20 succeeded" />
+)
+
+export const getTriggerStatusIcon = (triggerDetailStatus: string): JSX.Element => {
```

Comment:
Have picked the logic from earlier renderStatus and added few cases for enhancement